src/main/main.go

```diff
@@ -16,10 +16,20 @@ func main() {
 	config.ProcessArgs()
 	logger := utils.InitLogger(config.GlobalConfig.Verbose)
 	logger.Infof("Assisted installer started. Configuration is:\n %+v", config.GlobalConfig)
+
+	ops := ops.NewOps(logger)
+	out, err := ops.ExecPrivilegeCommand(nil, "cat", config.GlobalConfig.PullSecretPath)
```

Comment:
Why do we need to execute shell command?
Can't we read from the file?

Reply:
agree. but we will need to mount it i think

Reply:
not sure why we are doing it in main. Maybe i miss something